packages/asset/contracts/AssetReveal.sol

```diff
@@ -18,11 +18,11 @@ contract AssetReveal is IAssetReveal, Initializable, ERC2771Handler, EIP712Upgra
     AuthValidator private authValidator;
 
     // mapping of creator to asset id to asset's reveal nonce
-    mapping(address => mapping(uint256 => uint16)) revealIds;
+    mapping(address => mapping(uint256 => uint16)) public revealIds;
 
     // mapping for showing whether a revealHash has been used
     // revealHashes are generated by the TSB backend from reveal burn events and are used for reveal minting
-    mapping(bytes32 => bool) revealHashesUsed;
+    mapping(bytes32 => bool) public revealHashesUsed;
```

Comment:
If you make this public then there is probably no need to keep this function `revealHashUsed`

Reply:
I thing having a getter function is good practice and we should let it be